### PR TITLE
Quest log initial

### PR DESCRIPTION
**Pull request by gucio321**
Hi there,
In this PR I'm adding features requested in #801 (quest log), so:
* first a small correction of d2resource
* quest log with:
   - close and quest description buttons
   - tabs, which we can switch
   - clickable quest icons, which depends on selected tab
   - quest name, which depends on selected quest icon and is translated to local language

### CODE REVIEW

File: d2common/d2resource/resource_paths.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ const (
 	HealthManaIndicator = "/data/global/ui/PANEL/hlthmana.DC6"
 	AddSkillButton      = "/data/global/ui/PANEL/level.DC6"
 	GoldMoveDialog      = "/data/global/ui/menu/dialogbackground.DC6"
-	WPTabs              = "/data/global/ui/menu/expwaygatetabs.dc6"
+	WPTabs              = "/data/global/ui/MENU/expwaygatetabs.dc6"
```
Comment (gravestench) [inline d2common/d2resource/resource_paths.go:113]: did this capitalization prevent the file from loading?

File: d2core/d2ui/button.go [hunk 2]
```diff
@@ -40,16 +40,24 @@ const (
...
-	ButtonTypeMinipanelParty     ButtonType = 24
-	ButtonTypeBuy                ButtonType = 25
-	ButtonTypeSell               ButtonType = 26
-	ButtonTypeRepair             ButtonType = 27
-	ButtonTypeRepairAll          ButtonType = 28
-	ButtonTypeLeftArrow          ButtonType = 29
-	ButtonTypeRightArrow         ButtonType = 30
-	ButtonTypeQuery              ButtonType = 31
-	ButtonTypeSquelchChat        ButtonType = 32
+	ButtonTypeQuestDescr         ButtonType = 23 // I suppose it should be also used in WP tabs
```
Comment (gravestench) [inline d2core/d2ui/button.go:43]: the waypoint tab constants can just be an alias to these constants later, i think this is okay.

File: d2core/d2ui/button.go [hunk 3]
```diff
@@ -74,6 +82,14 @@ const (
...
 )
 
+const (
+	tab1BaseFrame = 0
+	tab2BaseFrame = 2
+	tab3BaseFrame = 4
+	tab4BaseFrame = 6
+	tab5BaseFrame = 8
+)
+
```
Comment (gravestench) [inline d2core/d2ui/button.go:92]: you can use `iota` here
```golang
const (
	tab1BaseFrame = iota * 2
	tab2BaseFrame
	tab3BaseFrame
	tab4BaseFrame
	tab5BaseFrame
)
```

**Review by gravestench — APPROVED** [review]

File: d2common/d2resource/resource_paths.go (hunk 1 above)
Comment (gucio321) [reply d2common/d2resource/resource_paths.go:113]: right, it doesn't matter

**Pull request merged**